cypress/integration/regions_test/regions_export_test.js

```diff
@@ -0,0 +1,93 @@
+describe("Regions Export Test", function(){
+    beforeEach(function(){
+        cy.visit("/biotestmine/regions");
+        cy.get(".input-section").within(() => {
+            cy.contains("Show Example").click();
+            cy.get("textarea").should("include.text","MAL");
+            cy.intercept("POST","/biotestmine/service/query/results").as("regionSearch");
+            cy.get("button").filter(':contains("Search")').click();
+            cy.wait("@regionSearch");
+        })
+    });
+
+    it("can export results of region search in TAB", function(){
+        cy.window().document().then(function (doc) {
+            doc.addEventListener('click', () => {
+              setTimeout(function () { doc.location.reload() }, 5000)
+            })
+        })
```

Comment:
Did you try moving this into `beforeEach`? Would also be a good idea to add a comment with the link to where this is from (github issue I think), as well as comment that it's a workaround to avoid a page timeout failure.